inst/templates/content-reference-topic.html

```diff
@@ -19,16 +19,13 @@ <h1>{{{title}}}</h1>
     {{/usage}}
 
     {{#has_args}}
-    <h2 class="hasAnchor" id="arguments"><a class="anchor" href="#arguments"></a>Arguments</h2>
-    <table class="ref-arguments">
-    <colgroup><col class="name" /><col class="desc" /></colgroup>
+    <h2>Arguments</h2>
+    <dl class="row" id="ref-arguments">
     {{#arguments}}
-    <tr>
-      <th>{{{name}}}</th>
-      <td>{{{description}}}</td>
-    </tr>
+      <dt class="col-sm-3 text-truncate" id="ref-arguments name">{{{name}}}</dt>
```

Comment:
Could I please ask you to delete `text-truncate`?

On my site, this class prevents me from reading the documentation for the functions. Is there a good reason for it?